ssl/methods.c

```diff
@@ -11,209 +11,156 @@
 #include <openssl/objects.h>
 #include "ssl_locl.h"
 
-/*
+/*-
  * TLS/SSLv3 methods
  */
 
 IMPLEMENT_tls_meth_func(TLS_ANY_VERSION, 0, 0,
                         TLS_method,
                         ossl_statem_accept,
                         ossl_statem_connect, TLSv1_2_enc_data)
-
 #ifndef OPENSSL_NO_TLS1_2_METHOD
-IMPLEMENT_tls_meth_func(TLS1_2_VERSION, 0, SSL_OP_NO_TLSv1_2,
+    IMPLEMENT_tls_meth_func(TLS1_2_VERSION, 0, SSL_OP_NO_TLSv1_2,
```

Comment:
@levitte, no, I mean that I'm rebased on top of #1414 but openssl-format-source still recreates the format diff.

 Looking at #1414 now, I'm not sure how that's supposed to solve it? You probably need to add `tls_meth_func` *similarly to* #1414?